internal/client/githubclient/githubclient_test.go

```diff
@@ -437,11 +437,11 @@ func Test_githubClient_GetWorkflow(t *testing.T) {
 				return
 			}
 			if len(got) != len(test.want) {
-				t.Fatalf("GetWorkflow got %v, want %v", got, test.want)
+				t.Fatalf("GetWorkflow() got length doesn't match want length = %v, want %v", len(got), len(test.want))
 			}
-			for i := 0; i < len(got); i++ {
+			for i := range got {
 				if got[i].Name != test.want[i].Name {
-					t.Fatalf("GetWorkflow got %v, want %v", got[i].Name, test.want[i].Name)
+					t.Fatalf("GetWorkflow() got = %v, want %v", got, test.want)
```

Comment:
The old line looks more correct here `got[i].Name, test.want[i].Name)`

Reply:
those are the two things being compared

Reply:
Good idea, reverted back to the old error message.